test_runner/regress/test_fast_import.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from fixtures.fast_import import FastImport
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import VanillaPostgres, PgProtocol, PgBin
+from fixtures.port_distributor import PortDistributor
+
+
```

Comment:
Perhaps makes sense to make this an extension to `test_pgdata_import_smoke`, which has this comment currently:
```
    # We have a Postgres data directory now.
    # Make a localfs remote storage that looks like how after `fast_import` ran.
    # TODO: actually exercise fast_import here
```

Reply:
Just thought that it might not worth doing it all in the same test due to increased complexity -> more complicated debugging if something is failing. Right now it is using vanilla postgres pgdata as we do in fast import, so it should be enough for testing pageserver part.

Moved mine simple test to the same file, but will keep `test_pgdata_import_smoke` the same. Also added a todo to test full import flow separately, which will complement `test_pgdata_import_smoke` and `test_fast_import_binary`, but not complicated any of those.

Reply:
WDYT?

Reply:
Agreed in DM that I will keep `test_pgdata_import_smoke` as is and add another one with s3 + pageserver + fast import binary, but if you don't mind I would do it in https://github.com/neondatabase/neon/pull/10407 because it will anyway need something close